content/articles/implementation-of-pattern-matching-in-the-building-of-type-driven-algorithms/index.md

```diff
@@ -0,0 +1,270 @@
+### Introduction
+This article is going to explore various functionalities of the C# programming language and how it is used in the implementation of various concepts but to be more specific, the article covers the use of pattern matching.
+### Table of contents
+- Prerequisite
+- Pattern matching
+- Pattern designs
+- Toll calculations
+- Add in the cost of occupancy
+- Addition to the pricing
+- Conclusion
+- What next
```

Comment:
Check this link that will help you with creating your table of contents correctly.
https://stackoverflow.com/questions/11948245/markdown-to-create-pages-and-table-of-contents